test/src/main/resources/DeploymentSpecifics.properties

```diff
@@ -1,11 +1,11 @@
 #
 # In this file, all information is stored that are specific for this
-# instance of the AdapterFramework.
+# instance of the Frank!Framework.
 #
 # The values in this file override the values in AppConstants.properties
 
-# the name of this instance of the AdapterFramework
-instance.name=Ibis4Test
+# the name of this instance of the Frank!Framework
+instance.name=Frank4Test
```

Comment:
Gaat dit goed in de larva tests?

Reply:
Ik hoop het maar in Jenkins is de build nog niet klaar? Tenzij jij deze had afgetrapt. Was wel iets waarover ik twijfelde inderdaad.